modules/osd-create-cluster-ccs.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_create/creating-an-aws-cluster.adoc
+// * osd_cluster_create/creating-a-gcp-cluster.adoc
+
+ifeval::["{context}" == "osd-creating-a-cluster-on-aws"]
+:osd-on-aws:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-gcp"]
+:osd-on-gcp:
+endif::[]
+
+:_content-type: PROCEDURE
+ifdef::osd-on-aws[]
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on AWS with CCS
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+[id="osd-create-gcp-cluster-ccs_{context}"]
+= Creating a cluster on GCP with CCS
+endif::osd-on-gcp[]
+
+By using the Customer Cloud Subscription (CCS) billing model, you can create an {product-title} cluster in an existing 
+ifdef::osd-on-aws[]
+{AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+{GCP} 
+endif::osd-on-gcp[]
+account that you own.
+
+You must meet several prerequisites if you use the CCS model to deploy and manage {product-title} into your AWS account.
+
+.Prerequisites
+
+ifdef::osd-on-aws[]
+* You have configured your AWS account for use with {product-title}.
+* You have not deployed any services in your AWS account.
+* You have configured the AWS account quotas and limits that are required to support the desired cluster size.
+* You have an `osdCcsAdmin` AWS Identity and Access Management (IAM) user with the `AdministratorAccess` policy attached.
+* You have set up a service control policy (SCP) in your AWS organization.
```

Comment:
UI shows `An Organization Service Control Policy (SCP) is set up according to the requirements for customer cloud subscriptions.` better to keep consistent. And if there is a doc ULR about SCP policy requirement will be better.

Reply:
In the AWS documentation, this is referred to as a "service control policy". I prefer to add "in your organization" rather than use "organizational" in the docs. I think it is more explicit.

That's a great idea about adding a link. Because we cannot include cross-references in modules, I have subsequently updated the prerequisite to the following:

"* You have set up a service control policy (SCP) in your AWS organization. For information, see Minimum required service control policy (SCP)."

I have also added an additional resources link to that section at the end of the assembly, so that the string and a link can be found on the same page.